modules/xfem/doc/content/source/actions/XFEMAction.md

```diff
@@ -1 +1,20 @@
-!template load file=stubs/moose_action.md.template name=XFEMAction syntax=/XFEM/XFEMAction
\ No newline at end of file
+# XFEM Action
+
+!syntax description /XFEM/XFEMAction
+
+## Overview
```

Comment:
I would refrain from saying much of anything in this page. Just point to the syntax page.  Look at TensorMechanicsAction.md as a template for how to do this.